test/notifications/display_test.dart

```diff
@@ -114,7 +130,11 @@ void main() {
       required String expectedTitle,
       required String expectedTagComponent,
       required bool expectedIsGroupConversation,
+      List<int>? expectedIconBitmap = kSolidBlueAvatar,
     }) {
+      assert(messageStyleMessages.every((e) => e.userId == data.userId
+                                            && e.realmUri == data.realmUri));
+
```

Comment:
oh.. I didn't catch this one before!
```suggestion
      assert(messageStyleMessages.every((e) => e.userId == data.userId
        && e.realmUri == data.realmUri));

```

Reply:
Separated the cases in separate `assert`s, since input list won't be large and it also matches [existing](https://github.com/zulip/zulip-flutter/blob/2393a193f54ae1411b5dc674cc03fd5adaa4ddea/test/example_data.dart#L423-L424) code.